src/js/expression-operand.js

```diff
@@ -43,13 +43,22 @@ angular.module('ngEquation')
         };
 
         ctrl.editMetadata = function() {
+            var valueInitialized = angular.isDefined(ctrl.options.value);
```

Comment:
when we discussed this in person, I envisioned `valueInitialized` being set to true when `angular.isDefined(result)`, rather than being updated every time `editMetadata` is called. what do you think of that approach?